src/wp-includes/rest-api/endpoints/class-wp-rest-block-patterns-controller.php

```diff
@@ -248,6 +249,12 @@ public function get_item_schema() {
 					'readonly'    => true,
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'template_types' => array(
+					'description' => __( 'An array of template types where the pattern fits.', 'gutenberg' ),
```

Comment:
```suggestion
					'description' => __( 'An array of template types where the pattern fits.' ),
```

Text domain is not needed for WP core